srcpkgs/NetworkManager/template

```diff
@@ -52,6 +53,9 @@ case "$XBPS_TARGET_MACHINE" in
 	*-musl)
 			CFLAGS+=" -DHAVE_SECURE_GETENV -Dsecure_getenv=getenv"
 			CFLAGS+=" -D__USE_POSIX199309 -DRTLD_DEEPBIND=0"
+			pre_build() {
+				patch -p0 < "${FILESDIR}"/fix-musl-breaking-glibc-include.patch
+			}
```

Comment:
Try to avoid conditional patches. This patch for example doesn't need to be explicitly for musl. Also patching should happen in post_extract().

Reply:
Okay, I'll do it in post_extract, but as far as I can see the build breaks the build with glibc